Makefile

```diff
@@ -100,7 +100,7 @@ server-linux-package-docker:
 	rm -rf package
 
 generate: ## Install and run code generators.
-	cd server; go get github.com/golang/mock/mockgen
+	cd server; go install github.com/golang/mock/mockgen@v1.6.0
```

Comment:
the install instructions changed for recent golang versions https://github.com/golang/mock#go-116